src/components/button/docgenInfo.json

```diff
@@ -155,6 +183,11 @@
       "description": "",
       "displayName": "renderChildren",
       "methods": []
+    },
+    {
+      "description": "",
+      "displayName": "Button",
+      "methods": []
     }
   ]
 }
```

Comment:
do we still need this file?

Reply:
I've created a separate ticket to remove the `docgen` stuff and update the story  to `csf`